### PR TITLE
Try to implement possibliluty to switch language

### PR DESCRIPTION
**Pull request by Ghislain1**

### CODE REVIEW

**Comment by idrice24** [comment]
Please any reference for this???

-----Message d'origine-----
De : "Ghislain" <notifications@github.com>
Envoyé : ‎22/‎12/‎2020 19:52
À : "idrice24/mns" <mns@noreply.github.com>
Cc : "idrice24" <idrice.tsafouet@yahoo.com>; "Review requested" <review_requested@noreply.github.com>
Objet : [idrice24/mns] Try to implement possibliluty to switch language (#418)



You can view, comment on, or merge this pull request online at:
  https://github.com/idrice24/mns/pull/418
Commit Summary
#417 Add the localize package. 
#417 Prepare templates for translations 
#417 Create model to represents country and local id accordig angular theory 
#417 Create Service to retrieve language 
#417 Modified html to provide possibilty to set alnaguage 
#417 FIXED error Modified html to provide possibilty to set language 
File Changes
M angular.json (8) 
M package-lock.json (221) 
M package.json (1) 
M src/app/app.component.html (2) 
M src/app/core/components/home/home.component.html (2) 
M src/app/shared/components/footer/footer.component.html (16) 
M src/app/shared/components/footer/footer.component.ts (29) 
A src/app/shared/models/app-language.ts (9) 
A src/app/shared/services/app-language.service.spec.ts (16) 
A src/app/shared/services/app-language.service.ts (23) 
M src/app/shared/shared.module.ts (3) 
M src/polyfills.ts (2) 
Patch Links:
https://github.com/idrice24/mns/pull/418.patch 
https://github.com/idrice24/mns/pull/418.diff

**Comment by Ghislain1** [comment]
REF: https://angular.io/guide/i18n  ----> For theory --> @idrice24  READ This  i going to ask you question   ---> Comments my PR if you don t understand some thing ...

I' m still looking for angular project which  also implements that

**Review by idrice24 — APPROVED** [review]
what is the use of 'schematics' here??

File: src/app/app.component.html [hunk 1]
```diff
@@ -5,6 +5,8 @@
   <div class="container-fluid" [@routeAnimations]="prepareRoute(outlet)">
```
Comment (idrice24) [inline src/app/app.component.html:5]: what is the rule of this #outlet

Comment (Ghislain1) [reply src/app/app.component.html:5]: #<Name> In  Angular means  assigning html element to variable here outlet

Comment (Ghislain1) [reply src/app/app.component.html:5]: See also #dv in post-list html file for  grid element or list element

**Comment by Ghislain1** [comment]
Check your Job-Email as soon as possible  👎

**Pull request merged**